spec/runtime/utils/SymbolicInputStream.lsl

```diff
@@ -0,0 +1,21 @@
+libsl "1.1.0";
+
+library std
+    version "11"
+    language "Java"
+    url "-";
+
+// imports
+
+import java/io/InputStream;
+
+
+// primary semantic types
+
+@GenerateMe
+@extends("java.io.InputStream")
+@public @final type SymbolicInputStream
+    is runtime.utils.SymbolicInputStream
```

Comment:
the prototype of which class is this class ? (I can't find such class "runtime.utils.SymbolicInputStream")

Reply:
I can move it into `libsl.utils.*` package for better clarity if needed.

Reply:
@dog-m Yes, this name will be better )